src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,10 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+    public Color getRandomColor() {
+        int index = new Random().nextInt(Color.values().length);
+        return Color.values()[index];
     }
```

Comment:
In this case, you create an instance of Random each time you call this method. Declare instances at the class field level and initialize in the field or the constructor

```suggestion
    private final Random random;
    
    public ColorSuplier(Random random) {
      this.random = new Random();
    }
    public Color getRandomColor() {
        int index = random.nextInt(Color.values().length);
        return Color.values()[index];
    }
```